### ISSUE TITLE
yolo.weights file usage

### ISSUE THREAD

**Issue by chengs2000** [thtrieu/darkflow#206]
issue #49 mentions that to classify 9000 categories it needs yolo9000 weight file. Then my question is how many categories the yolo.weights can be used for? 

If I remember correctly, yolo.weights is use for 20 categories. 
 
Can the checkpoint file generated with one model (say classify 2 categories) be used for another model (say classify 3 categories, one more category than the above model?

**Comment by placentian** [comment]
Just curious which cfg file should be used with yolo.weights on the google drive? I was not able to use  cfg/yolo.cfg with this weight file.

**Comment by adamzjk** [comment]
same issue, seems no cfg files could match weight files in google drive.

**Comment by adamzjk** [comment]
@placentian Update to the latest version of darkflow, and use the default cgf file `yolo.cfg` in cfg folder, this works for me.

**Comment by placentian** [comment]
@adamzjk thanks a lot for your message. I used the default yolo.cfg in cfg folder with the yolo.weights file downloaded from here:
https://drive.google.com/drive/folders/0B1tW_VtY7onidEwyQ2FtQVplWEU

I made sure I used the latest code here but still observe the problem. 

I did not have any problem when using other four weight files on google drive.

BTW: how much performance improvement we expect out of v2.0 yolo compared with v2.0 tiny-yolo-voc?
